etc/module.xml

```diff
@@ -2,7 +2,7 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
   <module name="NS8_Protect"
-    setup_version="2.4.100">
+    setup_version="2.4.101">
```

Comment:
... that if we need this to match what's in `composer.json` we should update the CircleCI config. Otherwise, as soon as this gets deployed to production the version numbers will be out of sync.

Reply:
The versions need to get incremented for now so that cruz can easily update

Reply:
We've had to create a `sdkdev` branch to work around this version bumping stuff (so we can require the `dev` branch of `protect-sdk-php` as well), so if we merge stuff to there then it should just work when Cruz goes to install it.